packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -422,6 +464,22 @@ const AccountTransferForm = observer(
             let hint_text;
             if (is_migration_status_present) {
                 hint_text = <Localize i18n_default_text='You can no longer open new positions with this account.' />;
+            } else if (is_unavailable_status_present) {
+                hint_text = (
+                    <Localize i18n_default_text='The server is temporarily unavailable for this account. We’re working to resolve this.' />
+                );
+            } else if (is_maintenance_status_present) {
+                const getMaintenanceTime = () => {
+                    if (selected_to.is_dxtrade) return '08:00 GMT';
```

Comment:
Shouldn't we remove the hard coded maintenance time until we receive it from the API ?